### PR TITLE
fixed timezone diff

### PR DESCRIPTION
**Pull request by sarthyparty**

### CODE REVIEW

**Comment by sarthyparty** [comment]
SW 362

**Review by jrvollmer — CHANGES_REQUESTED** [review]
A few points (second one is my fault):
- Rather than setting a static time zone, would we be able to match the host's time zone when they run the image?
- Sorry, I force pushed some versioning changes to main (last time I fix history on main, I swear). When you rebase off of main to update your branch, make sure you do it interactively and drop the "Formatting and refactoring for tag addition" commit.
- Once everything is working and ready to be merged into main, can you make a final separate commit named something like "Updated version for Docker timestamp bug fix" that only includes changing the version in `chase-car-dashboard/package.json` and `chase-car-dashboard/package-lock.json` to `1.1.3`? Running `npm i` after changing the `chase-car-dashboard/pakage.json` version will update the version in `chase-car-dashboard/package-lock.json`.

File: windows-pull-run.bat [hunk 1]
```diff
@@ -110,14 +112,22 @@ IF NOT "%1"=="" (
...
...AIgB9ACwAewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIAVQBUAEMALQAxADEAIgAsACIATABpAG4A dQB4ACIAOgAiAEUAdABjAC8ARwBNAFQAKwAxADEAIgB9ACwAewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIAQQBsAGUAdQB0AGkAYQBuACAAUwB0AGEAbgBkAGEAcgBkACAA VABpAG0AZQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAQQBtAGUAcgBpAGMAYQAvAEEAZABhAGsAIgB9ACwAewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIASABhAHcAYQBpAGkA YQBuACAAUwB0AGEAbgBkAGEAcgBkACAAVABpAG0AZQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAUABhAGMAaQBmAGkAYwAvAEgAbwBuAG8AbAB1AGwAdQAiAH0ALAB7ACIA VwBpAG4AZABvAHcAcwAiADoAIgBNAGEAcgBxAHUAZQBzAGEAcwAgAFMAdABhAG4AZABhAHIAZAAgAFQAaQBtAGUAIgAsACIATABpAG4AdQB4ACIAOgAiAFAAYQBjAGkA ZgBpAGMALwBNAGEAcgBxAHUAZQBzAGEAcwAiAH0ALAB7ACIAVwBpAG4AZABvAHcAcwAiADoAIgBBAGwAYQBzAGsAYQBuACAAUwB0AGEAbgBkAGEAcgBkACAAVABpAG0A ZQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAQQBtAGUAcgBpAGMAYQAvAEEAbgBjAGgAbwByAGEAZwBlACIAfQAsAHsAIgBXAGkAbgBkAG8AdwBzACIAOgAiAFUAVABDAC0A MAA5ACIALAAiAEwAaQBuAHUAeAAiADoAIgBFAHQAYwAvAEcATQBUACsAOQAiAH0ALAB7ACIAVwBpAG4AZABvAHcAcwAiADoAIgBQAGEAYwBpAGYAaQBjACAAUwB0AGEA bgBkAGEAcgBkACAAVABpAG0AZQAgACgATQBlAHgAaQBjAG8AKQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAQQBtAGUAcgBpAGMAYQAvAFQAaQBqAHUAYQBuAGEAIgB9ACwA ewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIAVQBUAEMALQAwADgAIgAsACIATABpAG4AdQB4ACIAOgAiAEUAdABjAC8ARwBNA`
```
Comment (jrvollmer) [inline windows-pull-run.bat]: Can you guys please have the PowerShell file separate and spawn that here? I can't really review this line, and it's not maintainable

**Review by jrvollmer — CHANGES_REQUESTED** [review]

**Comment by jackwherry** [comment]
Here's the PowerShell (on multiple lines, I did my best to sorta reformat it):
```PowerShell
 $tzstring = '[{"Windows":"Dateline Standard Time","Linux":"Etc/GMT+12"},{"Windows":"UTC-11","Linux":"Etc/GMT+11"},{"Windows":"Aleutian Standard Time","Linux":"America/Adak"},{"Windows":"Hawaiian Standard Time","Linux":"Pacific/Honolulu"},{"Windows":"Marquesas Standard Time","Linux":"Pacific/Marquesas"},{"Windows":"Alaskan Standard Time","Linux":"America/Anchorage"},{"Windows":"UTC-09","Linux":"Etc/GMT+9"},{"Windows":"Pacific Standard Time (Mexico)","Linux":"America/Tijuana"},{"Windows":"UTC-08","Linux":"Etc/GMT+8"},{"Windows":"Pacific Standard Time","Linux":"America/Los_Angeles"},{"Windows":"US Mountain Standard Time","Linux":"America/Phoenix"},{"Windows":"Mountain Standard Time (Mexico)","Linux":"America/Chihuahua"},{"Windows":"Mountain Standard Time","Linux":"America/Denver"},{"Windows":"Central America Standard Time","Linux":"America/Guatemala"},{"Windows":"Central Standard Time","Linux":"America/Chicago"},{"Windows":"Easter Island Standard Time","Linux":"Pacific/Easter"},{"Windows":"Central Standard Time (Mexico)","Linux":"America/Mexico_City"},{"Windows":"Canada Central Standard Time","Linux":"America/Regina"},{"Windows":"SA Pacific Standard Time","Linux":"America/Bogota"},{"Windows":"Eastern Standard Time (Mexico)","Linux":"America/Cancun"},{"Windows":"Eastern Standard Time","Linux":"America/New_York"},{"Windows":"Haiti Standard Time","Linux":"America/Port-au-Prince"},{"Windows":"Cuba Standard Time","Linux":"America/Havana"},{"Windows":"US Eastern Standard Time","Linux":"America/Indiana/Indianapolis"},{"Windows":"Turks And Caicos Standard Time","Linux":"America/Grand_Turk"},{"Windows":"Paraguay Standard Time","Linux":"America/Asuncion"},{"Windows":"Atlantic Standard Time","Linux":"America/Halifax"},{"Windows":"Venezuela Standard Time","Linux":"America/Caracas"},{"Windows":"Central Brazilian Standard Time","Linux":"America/Cuiaba"},{"Windows":"SA Western Standard Time","Linux":"America/La_Paz"},{"Windows":"Pacific SA Standard Time","Linux":"America/Santiago"},{"Windows":"Newfoundland Standard Time","Linux":"America/St_Johns"},{"Windows":"Tocantins Standard Time","Linux":"America/Araguaina"},{"Windows":"E. South America Standard Time","Linux":"America/Sao_Paulo"},{"Windows":"SA Eastern Standard Time","Linux":"America/Cayenne"},{"Windows":"Argentina Standard Time","Linux":"America/Argentina/Buenos_Aires"},{"Windows":"Greenland Standard Time","Linux":"America/Godthab"},{"Windows":"Montevideo Standard Time","Linux":"America/Montevideo"},{"Windows":"Magallanes Standard Time","Linux":"America/Punta_Arenas"},{"Windows":"Saint Pierre Standard Time","Linux":"America/Miquelon"},{"Windows":"Bahia Standard Time","Linux":"America/Bahia"},{"Windows":"UTC-02","Linux":"Etc/GMT+2"},{"Windows":"Mid-Atlantic Standard Time","Linux":"Etc/GMT+2"},{"Windows":"Azores Standard Time","Linux":"Atlantic/Azores"},{"Windows":"Cape Verde Standard Time","Linux":"Atlantic/Cape_Verde"},{"Windows":"UTC","Linux":"Etc/UTC"},{"Windows":"Morocco Standard Time","Linux":"Africa/Casablanca"},{"Windows":"GMT Standard Time","Linux":"Europe/London"},{"Windows":"Greenwich Standard Time","Linux":"Atlantic/Reykjavik"},{"Windows":"W. Europe Standard Time","Linux":"Europe/Berlin"},{"Windows":"Central Europe Standard Time","Linux":"Europe/Budapest"},{"Windows":"Romance Standard Time","Linux":"Europe/Paris"},{"Windows":"Sao Tome Standard Time","Linux":"Africa/Sao_Tome"},{"Windows":"Central European Standard Time","Linux":"Europe/Warsaw"},{"Windows":"W. Central Africa Standard Time","Linux":"Africa/Lagos"},{"Windows":"Jordan Standard Time","Linux":"Asia/Amman"},{"Windows":"GTB Standard Time","Linux":"Europe/Bucharest"},{"Windows":"Middle East Standard Time","Linux":"Asia/Beirut"},{"Windows":"Egypt Standard Time","Linux":"Africa/Cairo"},{"Windows":"E. Europe Standard Time","Linux":"Europe/Chisinau"},{"Windows":"Syria Standard Time","Linux":"Asia/Damascus"},{"Windows":"West Bank Standard Time","Linux":"Asia/Hebron"},{"Windows":"South Africa Standard Time","Linux":"Africa/Johannesburg"},{"Windows":"FLE Standard Time","Linux":"Europe/Kiev"},{"Windows":"Israel Standard Time","Linux":"Asia/Jerusalem"},{"Windows":"Kaliningrad Standard Time","Linux":"Europe/Kaliningrad"},{"Windows":"Sudan Standard Time","Linux":"Africa/Khartoum"},{"Windows":"Libya Standard Time","Linux":"Africa/Tripoli"},{"Windows":"Namibia Standard Time","Linux":"Africa/Windhoek"},{"Windows":"Arabic Standard Time","Linux":"Asia/Baghdad"},{"Windows":"Turkey Standard Time","Linux":"Europe/Istanbul"},{"Windows":"Arab Standard Time","Linux":"Asia/Riyadh"},{"Windows":"Belarus Standard Time","Linux":"Europe/Minsk"},{"Windows":"Russian Standard Time","Linux":"Europe/Moscow"},{"Windows":"E. Africa Standard Time","Linux":"Africa/Nairobi"},{"Windows":"Iran Standard Time","Linux":"Asia/Tehran"},{"Windows":"Arabian Standard Time","Linux":"Asia/Dubai"},{"Windows":"Astrakhan Standard Time","Linux":"Europe/Astrakhan"},{"Windows":"Azerbaijan Standard Time","Linux":"Asia/Baku"},{"Windows":"Russia Time Zone 3","Linux":"Europe/Samara"},{"Windows":"Mauritius Standard Time","Linux":"Indian/Mauritius"},{"Windows":"Saratov Standard Time","Linux":"Europe/Saratov"},{"Windows":"Georgian Standard Time","Linux":"Asia/Tbilisi"},{"Windows":"Caucasus Standard Time","Linux":"Asia/Yerevan"},{"Windows":"Afghanistan Standard Time","Linux":"Asia/Kabul"},{"Windows":"West Asia Standard Time","Linux":"Asia/Tashkent"},{"Windows":"Ekaterinburg Standard Time","Linux":"Asia/Yekaterinburg"},{"Windows":"Pakistan Standard Time","Linux":"Asia/Karachi"},{"Windows":"India Standard Time","Linux":"Asia/Kolkata"},{"Windows":"Sri Lanka Standard Time","Linux":"Asia/Colombo"},{"Windows":"Nepal Standard Time","Linux":"Asia/Kathmandu"},{"Windows":"Central Asia Standard Time","Linux":"Asia/Almaty"},{"Windows":"Bangladesh Standard Time","Linux":"Asia/Dhaka"},{"Windows":"Omsk Standard Time","Linux":"Asia/Omsk"},{"Windows":"Myanmar Standard Time","Linux":"Asia/Yangon"},{"Windows":"SE Asia Standard Time","Linux":"Asia/Bangkok"},{"Windows":"Altai Standard Time","Linux":"Asia/Barnaul"},{"Windows":"W. Mongolia Standard Time","Linux":"Asia/Hovd"},{"Windows":"North Asia Standard Time","Linux":"Asia/Krasnoyarsk"},{"Windows":"N. Central Asia Standard Time","Linux":"Asia/Novosibirsk"},{"Windows":"Tomsk Standard Time","Linux":"Asia/Tomsk"},{"Windows":"China Standard Time","Linux":"Asia/Shanghai"},{"Windows":"North Asia East Standard Time","Linux":"Asia/Irkutsk"},{"Windows":"Singapore Standard Time","Linux":"Asia/Singapore"},{"Windows":"W. Australia Standard Time","Linux":"Australia/Perth"},{"Windows":"Taipei Standard Time","Linux":"Asia/Taipei"},{"Windows":"Ulaanbaatar Standard Time","Linux":"Asia/Ulaanbaatar"},{"Windows":"North Korea Standard Time","Linux":"Asia/Pyongyang"},{"Windows":"Aus Central W. Standard Time","Linux":"Australia/Eucla"},{"Windows":"Transbaikal Standard Time","Linux":"Asia/Chita"},{"Windows":"Tokyo Standard Time","Linux":"Asia/Tokyo"},{"Windows":"Korea Standard Time","Linux":"Asia/Seoul"},{"Windows":"Yakutsk Standard Time","Linux":"Asia/Yakutsk"},{"Windows":"Cen. Australia Standard Time","Linux":"Australia/Adelaide"},{"Windows":"AUS Central Standard Time","Linux":"Australia/Darwin"},{"Windows":"E. Australia Standard Time","Linux":"Australia/Brisbane"},{"Windows":"AUS Eastern Standard Time","Linux":"Australia/Sydney"},{"Windows":"West Pacific Standard Time","Linux":"Pacific/Port_Moresby"},{"Windows":"Tasmania Standard Time","Linux":"Australia/Hobart"},{"Windows":"Vladivostok Standard Time","Linux":"Asia/Vladivostok"},{"Windows":"Lord Howe Standard Time","Linux":"Australia/Lord_Howe"},{"Windows":"Bougainville Standard Time","Linux":"Pacific/Bougainville"},{"Windows":"Russia Time Zone 10","Linux":"Asia/Srednekolymsk"},{"Windows":"Magadan Standard Time","Linux":"Asia/Magadan"},{"Windows":"Norfolk Standard Time","Linux":"Pacific/Norfolk"},{"Windows":"Sakhalin Standard Time","Linux":"Asia/Sakhalin"},{"Windows":"Central Pacific Standard Time","Linux":"Pacific/Guadalcanal"},{"Windows":"Russia Time Zone 11","Linux":"Asia/Kamchatka"},{"Windows":"New Zealand Standard Time","Linux":"Pacific/Auckland"},{"Windows":"UTC+12","Linux":"Etc/GMT-12"},{"Windows":"Fiji Standard Time","Linux":"Pacific/Fiji"},{"Windows":"Kamchatka Standard Time","Linux":"Asia/Kamchatka"},{"Windows":"Chatham Islands Standard Time","Linux":"Pacific/Chatham"},{"Windows":"UTC+13","Linux":"Etc/GMT-13"},{"Windows":"Tonga Standard Time","Linux":"Pacific/Tongatapu"},{"Windows":"Samoa Standard Time","Linux":"Pacific/Apia"},{"Windows":"Line Islands Standard Time","Linux":"Pacific/Kiritimati"}]';

function ConvertTo-Hashtable { 
    [CmdletBinding()] 
    [OutputType('hashtable')] 
    param ( [Parameter(ValueFromPipeline)] $InputObject )
        process { 
        if ($null -eq $InputObject) { return $null; } 
        if ($InputObject -is [System.Collections.IEnumerable] -and $InputObject -isnot [string]) { 
            $collection = @( foreach ($object in $InputObject) { ConvertTo-Hashtable -InputObject $object; } ); 
            Write-Output -NoEnumerate $collection 
        } elseif ($InputObject -is [psobject]) { 
            $hash = @{}; foreach ($property in $InputObject.PSObject.Properties) { $hash[$property.Name] = ConvertTo-Hashtable -InputObject $property.Value; } $hash; 
        } else { $InputObject; } 
    } 
} 

$tzarr = $tzstring | ConvertFrom-Json | ConvertTo-HashTable; 
$tz = tzutil /g; 
$linuxtz = ""; 
foreach( $tzpair in $tzarr ) { 
    if ( $tzpair.Get_Item("Windows") -eq $tz ) { 
        $linuxtz = $tzpair.Get_Item("Linux"); break; 
    } 
} 
Write-Host $linuxtz; 
```

File: linux-pull-run.sh [hunk 2]
```diff
@@ -76,21 +76,27 @@ if [[ "$1" == '--' ]]; then shift; fi
 tag=${tag:=latest}
 
 # Pull the Docker image with the specified tag
-docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
-
+#docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
+# == darwin signifies mac
```
Comment (jrvollmer) [inline linux-pull-run.sh]:
```suggestion
docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
```

File: linux-pull-run.sh [hunk 3]
```diff
@@ -76,21 +76,27 @@ if [[ "$1" == '--' ]]; then shift; fi
...
+	else
+		echo -e "\nCould not run \`$cmd\`, please open 'http://localhost:3000' on your browser\n"
+	fi
 fi
 
+# Setting the time zone of host machine
+[[ $OSTYPE == 'darwin'* ]] && timezone=`readlink /etc/localtime | sed 's#/var/db/timezone/zoneinfo/##g'`
+[[ $OSTYPE != 'darwin'* ]] && timezone=`cat /etc/timezone`
+  #timezone="date +%z"
+
```
Comment (jrvollmer) [inline linux-pull-run.sh]:
```suggestion
```

File: windows-pull-run.bat [hunk 4]
```diff
@@ -6,6 +6,7 @@ setlocal enabledelayedexpansion
 SET exitcode=0
 
 GOTO loop
+SET timezone1=`tzutil /g`
```
Comment (jrvollmer) [inline windows-pull-run.bat]:
```suggestion
```
If we aren't using this variable, we should removed it.

File: windows-pull-run.bat [hunk 5]
```diff
@@ -110,14 +112,23 @@ IF NOT "%1"=="" (
...
...AIgB9ACwAewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIAVQBUAEMALQAxADEAIgAsACIATABpAG4A dQB4ACIAOgAiAEUAdABjAC8ARwBNAFQAKwAxADEAIgB9ACwAewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIAQQBsAGUAdQB0AGkAYQBuACAAUwB0AGEAbgBkAGEAcgBkACAA VABpAG0AZQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAQQBtAGUAcgBpAGMAYQAvAEEAZABhAGsAIgB9ACwAewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIASABhAHcAYQBpAGkA YQBuACAAUwB0AGEAbgBkAGEAcgBkACAAVABpAG0AZQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAUABhAGMAaQBmAGkAYwAvAEgAbwBuAG8AbAB1AGwAdQAiAH0ALAB7ACIA VwBpAG4AZABvAHcAcwAiADoAIgBNAGEAcgBxAHUAZQBzAGEAcwAgAFMAdABhAG4AZABhAHIAZAAgAFQAaQBtAGUAIgAsACIATABpAG4AdQB4ACIAOgAiAFAAYQBjAGkA ZgBpAGMALwBNAGEAcgBxAHUAZQBzAGEAcwAiAH0ALAB7ACIAVwBpAG4AZABvAHcAcwAiADoAIgBBAGwAYQBzAGsAYQBuACAAUwB0AGEAbgBkAGEAcgBkACAAVABpAG0A ZQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAQQBtAGUAcgBpAGMAYQAvAEEAbgBjAGgAbwByAGEAZwBlACIAfQAsAHsAIgBXAGkAbgBkAG8AdwBzACIAOgAiAFUAVABDAC0A MAA5ACIALAAiAEwAaQBuAHUAeAAiADoAIgBFAHQAYwAvAEcATQBUACsAOQAiAH0ALAB7ACIAVwBpAG4AZABvAHcAcwAiADoAIgBQAGEAYwBpAGYAaQBjACAAUwB0AGEA bgBkAGEAcgBkACAAVABpAG0AZQAgACgATQBlAHgAaQBjAG8AKQAiACwAIgBMAGkAbgB1AHgAIgA6ACIAQQBtAGUAcgBpAGMAYQAvAFQAaQBqAHUAYQBuAGEAIgB9ACwA ewAiAFcAaQBuAGQAbwB3AHMAIgA6ACIAVQBUAEMALQAwADgAIgAsACIATABpAG4AdQB4ACIAOgAiAEUAdABjAC8ARwBNA`
```
Comment (jrvollmer) [inline windows-pull-run.bat]:
```suggestion
```

File: windows-pull-run.bat [hunk 6]
```diff
@@ -110,14 +112,23 @@ IF NOT "%1"=="" (
...
+for /f %%i in ('powershell -executionpolicy remotesigned -File .\convertTZstamp.ps1') do set timezone2=%%i
+
+@REM FOR /F "tokens=*" %%g IN ('tzutil /g') do (SET timezone=%%g)
+
+@REM TODO Map timezone to Docker friendly format (e.g. America/Chicago) using a Python script
```
Comment (jrvollmer) [inline windows-pull-run.bat]:
```suggestion
```

File: windows-pull-run.bat [hunk 7]
```diff
@@ -110,14 +112,23 @@ IF NOT "%1"=="" (
...
+for /f %%i in ('powershell -executionpolicy remotesigned -File .\convertTZstamp.ps1') do set timezone2=%%i
```
Comment (jrvollmer) [inline windows-pull-run.bat]:
```suggestion
for /f %%i in ('powershell -executionpolicy remotesigned -File .\convertTZstamp.ps1') do set timezone=%%i
```
Only one timezone variable is being used

File: windows-pull-run.bat [hunk 8]
```diff
@@ -110,14 +112,23 @@ IF NOT "%1"=="" (
...
-echo docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
+docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
 
 @REM Open the dashboard if no open wasn't specified
-IF NOT "%no_open%"=="true" echo start http://localhost:3000
+IF NOT "%no_open%"=="true" start http://localhost:3000
 
 @REM Run the dashboard image
-echo docker run -p 3000:3000 ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
+docker run -e TZ=%timezone2% -p 3000:3000 ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
```
Comment (jrvollmer) [inline windows-pull-run.bat]:
```suggestion
docker run -e TZ=%timezone% -p 3000:3000 ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
```
Goes with previous comment about only having one timezone variable. This line was also overwritten, so there will be another suggestions in the next review

**Review by jrvollmer — CHANGES_REQUESTED** [review]
Suggestions for non-overwritten changes

File: linux-pull-run.sh [hunk 9]
```diff
@@ -89,13 +89,14 @@ if [[ ! $no_open ]]; then
 	fi
 fi
 
-# Create the Docker volume
-path=`pwd`
-mkdir -p recordedData
-docker volume create --name chasecar --opt type=none --opt device=${path}/recordedData --opt o=bind
+# Setting the time zone of host machine
```
Comment (jrvollmer) [inline linux-pull-run.sh:97]:
```suggestion
# Create the Docker volume
path=`pwd`
mkdir -p recordedData
docker volume create --name chasecar --opt type=none --opt device=${path}/recordedData --opt o=bind

# Setting the time zone of host machine
```

File: README.md [hunk 10]
```diff
@@ -36,9 +36,9 @@
...
-   - (Windows) Navigate to the folder storing the script and double click on it. 
-   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh` and `./linux-pull-run.sh`.
+4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it to your Downloads folder. If operating on Windows make sure to remove the ".txt" when downloading and also download the powershell script in same folder To run, look at the bullet points below:
+   - (Windows) Navigate to the Downloads folder and double click on the script. 
+   - (MacOS/Linux) Run these commands in your terminal `cd Downloads`, `chmod +x linux-pull-run.sh`, and `./linux-pull-run.sh`.
```
Comment (jrvollmer) [inline README.md]:
```suggestion
4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it. This script will create a folder called recordedData in the same spot as your script, so keep that in mind when choosing where to download it. If operating on Windows make sure to remove the ".txt" when downloading and also download the powershell script in same folder. To run, look at the bullet points below:
   - (Windows) Navigate to the folder storing the script and double click on it. 
   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh`, and `./linux-pull-run.sh`.
```

File: linux-pull-run.sh [hunk 11]
```diff
@@ -89,13 +89,14 @@ if [[ ! $no_open ]]; then
...
+[[ $OSTYPE == 'darwin'* ]] && timezone=`readlink /etc/localtime | sed 's#/var/db/timezone/zoneinfo/##g'`
+[[ $OSTYPE != 'darwin'* ]] && timezone=`cat /etc/timezone`
+  #timezone="date +%z"
+
 
 # Run the Docker image
-docker run -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
+docker run -e TZ=$timezone -p 3000:3000 ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
```
Comment (jrvollmer) [inline linux-pull-run.sh]:
```suggestion
docker run -e TZ=$timezone -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
```

File: windows-pull-run.bat [hunk 12]
```diff
@@ -122,7 +134,9 @@ mkdir recordedData
 docker volume create --name chasecar --opt type=none --opt device=%recdatapath%/recordedData --opt o=bind
 
 @REM Run the dashboard image
-docker run -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
+
+docker run -e TZ=%timezone2% -p 3000:3000 ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
+
```
Comment (jrvollmer) [inline windows-pull-run.bat]:
```suggestion
docker run -e TZ=%timezone% -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
```

**Review by jrvollmer — CHANGES_REQUESTED** [review]
Below are changes to make to address the lines that were overwritten when rebasing

File: windows-pull-run.bat [hunk 13]
```diff
@@ -122,7 +127,9 @@ mkdir recordedData
 docker volume create --name chasecar --opt type=none --opt device=%recdatapath%/recordedData --opt o=bind
 
 @REM Run the dashboard image
-docker run -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
+
+docker run -e TZ=%timezone% -p 3000:3000 ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
+
```
Comment (jrvollmer) [inline windows-pull-run.bat]:
```suggestion
docker run -e TZ=%timezone% -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
```

File: README.md [hunk 14]
```diff
@@ -36,9 +36,9 @@
...
+4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it. This script will create a folder called recordedData in the same spot as your script, so keep that in mind when choosing where to download it. If operating on Windows make sure to remove the ".txt" when downloading and also download the powershell script in same folder. To run, look at the bullet points below:
    - (Windows) Navigate to the folder storing the script and double click on it. 
-   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh` and `./linux-pull-run.sh`.
+   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh`, and `./linux-pull-run.sh`.
```
Comment (jrvollmer) [inline README.md]:
```suggestion
   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh` and `./linux-pull-run.sh`.
```

**Review by jrvollmer — CHANGES_REQUESTED** [review]

File: linux-pull-run.sh [hunk 15]
```diff
@@ -77,7 +77,6 @@ tag=${tag:=latest}
 
 # Pull the Docker image with the specified tag
 docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
```
Comment (jrvollmer) [inline linux-pull-run.sh]:
```suggestion
docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag

```

File: windows-pull-run.bat [hunk 16]
```diff
@@ -102,6 +102,7 @@ IF NOT "%1"=="" (
 	IF "!help!"=="true" (
 		GOTO printusage
 	)
+
```
Comment (jrvollmer) [inline windows-pull-run.bat]:
```suggestion
```

**Review by jrvollmer — CHANGES_REQUESTED** [review]
Two more quick changes to go with the two above

File: README.md [hunk 17]
```diff
@@ -36,9 +36,15 @@
...
    - (Windows) Navigate to the folder storing the script and double click on it. 
    - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh` and `./linux-pull-run.sh`.
+=======
+4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it. This script will create a folder called recordedData in the same spot as your script, so keep that in mind when choosing where to download it. If operating on Windows make sure to remove the ".txt" when downloading and also download the powershell script in same folder. To run, look at the bullet points below:
+   - (Windows) Navigate to the folder storing the script and double click on it. 
+   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh`, and `./linux-pull-run.sh`.
+>>>>>>> 10dd8a5 (after updating my pull testing-changes)
```
Comment (jrvollmer) [inline README.md]:
```suggestion
4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it. This script will create a folder called recordedData in the same spot as your script, so keep that in mind when choosing where to download it. If operating on Windows make sure to remove the ".txt" when downloading and also download the powershell script in same folder. To run, look at the bullet points below:
   - (Windows) Navigate to the folder storing the script and double click on it. 
   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh` and `./linux-pull-run.sh`.
```

File: linux-pull-run.sh [hunk 18]
```diff
@@ -94,8 +93,17 @@ path=`pwd`
...
+[[ $OSTYPE != 'darwin'* ]] && timezone=`cat /etc/timezone`
+
 # Run the Docker image
-docker run -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
+<<<<<<< HEAD
+docker run -e TZ=$timezone -p 3000:3000  -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
+
+=======
+docker run -e TZ=$timezone -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
+>>>>>>> 10dd8a5 (after updating my pull testing-changes)
```
Comment (jrvollmer) [inline linux-pull-run.sh]:
```suggestion
docker run -e TZ=$timezone -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
```

File: windows-pull-run.bat [hunk 19]
```diff
@@ -121,8 +126,9 @@ SET recdatapath=%cd%
 mkdir recordedData
 docker volume create --name chasecar --opt type=none --opt device=%recdatapath%/recordedData --opt o=bind
 
-@REM Run the dashboard image
-docker run -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
+docker run -e TZ=%timezone% -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
+
+
```
Comment (jrvollmer) [inline windows-pull-run.bat]:
```suggestion
@REM Run the dashboard image
docker run -e TZ=%timezone% -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:%tag%
```

**Review by jrvollmer — COMMENTED** [review]
Some miscellaneous stuff that was left after updating

**Review by jrvollmer — APPROVED** [review]

**Pull request merged**